### PR TITLE
Feature/dynamic tags teaser

### PR DESCRIPTION
**Pull request by udidol**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: assets/dev/js/editor/components/dynamic-tags/control-behavior.js [hunk 1]
```diff
@@ -72,6 +72,20 @@ module.exports = Marionette.Behavior.extend( {
...
 
+		// Create and inject pro dynamic teaser template if Pro is not installed
+		if ( ! window.elementorPro && Object.keys( tags ).length ) {
+			const $proTeaser = jQuery( '<div>', { class: 'elementor-tags-list__teaser' } ),
+				$proTeaserTitle = jQuery( '<div>', { class: 'elementor-tags-list__group-title' } )
+					.html( '<i class="eicon-info-circle"></i>Elementor Dynamic Content' ),
+				$proTeaserText = jQuery( '<div>', { class: 'elementor-tags-list__teaser-text' } )
+					.html( 'You’re missing out!<br />' +
+						'Get more dynamic capabilities by incorporating dozens of Elementor’s native dynamic tags. ' +
+						'<a href="https://go.elementor.com/go-pro-dynamic-tag" class="elementor-tags-list__teaser-link" target="_blank">See it in action</a>'
```
Comment (kobizz) [inline assets/dev/js/editor/components/dynamic-tags/control-behavior.js]: Retrieve the URL from config

File: assets/dev/scss/editor/_dynamic-tags.scss [hunk 2]
```diff
@@ -11,10 +11,17 @@
 	z-index: $first-layer;
 
 	&__group-title {
-		color: $editor-darkest;
+		color: $editor-dark;
```
Comment (kobizz) [inline assets/dev/scss/editor/_dynamic-tags.scss]: ?

File: assets/dev/scss/editor/_dynamic-tags.scss [hunk 3]
```diff
@@ -11,10 +11,17 @@
...
 		font-weight: bold;
 		font-size: 12px;
 		padding: 10px 15px;
+
+		i.eicon-info-circle {
```
Comment (kobizz) [inline assets/dev/scss/editor/_dynamic-tags.scss]: Omit the `i` tag

File: assets/dev/scss/editor/_dynamic-tags.scss [hunk 4]
```diff
@@ -11,10 +11,17 @@
...
+			@include padding-end(5px);
+			color: #71d7f7;
```
Comment (kobizz) [inline assets/dev/scss/editor/_dynamic-tags.scss]: Use `$editor-info` variable

File: assets/dev/scss/editor/_dynamic-tags.scss [hunk 5]
```diff
@@ -11,10 +11,17 @@
...
+			font-size: 14px;
+			vertical-align: text-bottom;
```
Comment (kobizz) [inline assets/dev/scss/editor/_dynamic-tags.scss]: Don't use this property

File: assets/dev/scss/editor/_dynamic-tags.scss [hunk 6]
```diff
@@ -32,6 +39,25 @@
 			background-color: $editor-background;
 		}
 	}
+
+	&__teaser {
+		border-top: 2px solid $editor-lightest;
+		padding-top: 4px;
+		margin-top: 2px;
```
Comment (kobizz) [inline assets/dev/scss/editor/_dynamic-tags.scss]: Let's discuss it

File: assets/dev/scss/editor/_dynamic-tags.scss [hunk 7]
```diff
@@ -32,6 +39,25 @@
...
+
+		&-text {
+			padding: 2px 15px 8px 15px;
```
Comment (kobizz) [inline assets/dev/scss/editor/_dynamic-tags.scss]: The last value is redundant

**Review by kobizz — CHANGES_REQUESTED** [review]

File: assets/dev/scss/editor/_dynamic-tags.scss (hunk 2 above)
Comment (udidol) [reply assets/dev/scss/editor/_dynamic-tags.scss]: was too dark, darker than the figma

**Pull request closed without merging**